sklearn/utils/tests/test_validation.py

```diff
@@ -141,6 +141,11 @@ def test_check_array():
     X_nan[0, 0] = np.nan
     assert_raises(ValueError, check_array, X_nan)
     check_array(X_inf, force_all_finite=False)  # no raise
+    # allow_nan check
+    check_array(X_nan, force_all_finite=True, allow_nan=True)  # no raise
```

Comment:
this is a bit hard to read, but I guess it's too late to rename ``force_all_finite``.